controllers/kogitoserverlessplatform_controller.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	swkogitov08 "github.com/davidesalerno/kogito-serverless-operator/api/v08"
+)
+
+// KogitoServerlessPlatformReconciler reconciles a KogitoServerlessPlatform object
+type KogitoServerlessPlatformReconciler struct {
+	client.Client
+	Scheme *runtime.Scheme
+}
+
+//+kubebuilder:rbac:groups=sw.kogito.kie.org,resources=kogitoserverlessplatforms,verbs=get;list;watch;create;update;patch;delete
```

Comment:
If I recall correctly these annotations are not use in the controller

Reply:
I think that they are used to generate the rbac configuration you can find here -> https://github.com/davidesalerno/kogito-serverless-operator/tree/main/config/rbac